web/src/lib/utils/helpers.ts

```diff
@@ -41,6 +41,19 @@ export const useMountEffectWhenDefined = (fun: () => void, thingToBeDefined: unk
   }, [thingToBeDefined, fun]);
 };
 
+export const useScrollToPathAnchor = () => {
+  useEffect(() => {
+    const path = window.location.hash;
+    if (path && path.includes("#")) {
+      const id = path.replace("#", "");
+      const element = document.querySelector(`#${id}`);
+      if (id && element) {
+        element.scrollIntoView({ behavior: "smooth" });
+      }
+    }
+  });
+};
+
```

Comment:
This is, for some unknown reason, necessary because of [this nextjs issue](https://github.com/vercel/next.js/issues/11109)

Without it, anchor links don't work from a URL on page load. they work if you're already on the page, but not on reload or when navigating from another page